batch/batch/cloud/azure/resource_utils.py

```diff
@@ -102,6 +102,12 @@ def azure_disk_from_storage_in_gib(disk_type: str, storage_in_gib: int) -> Optio
     return disks[index]
 
 
+def azure_disk_from_storage_in_bytes(disk_type: str, storage_in_bytes: int) -> Optional[AzureDisk]:
+    storage_in_gib = storage_in_bytes / 1024 / 1024 / 1024
+    storage_in_gib = math.ceil(storage_in_gib)
```

Comment:
ehhhh I don't like going through floating point for this kind of calculation. seems wrong when we can do this with integral arithmetic.

```
one_gib = 1024 * 1024 * 1024
storage_in_gib = (storage_in_bytes - 1 + one_gib) // one_gib
```